src/OpenApi/src/Services/OpenApiDocumentService.cs

```diff
@@ -38,6 +38,9 @@ internal sealed class OpenApiDocumentService(
     private readonly OpenApiSchemaService _componentService = serviceProvider.GetRequiredKeyedService<OpenApiSchemaService>(documentName);
     private readonly OpenApiSchemaReferenceTransformer _schemaReferenceTransformer = new();
 
+    private List<IOpenApiSchemaTransformer>? _schemaTransformers;
```

Comment:
Do null and empty have different meanings?  If not, would it be easier to initialize these to empty lists?

Reply:
Yeah...I think this was an unfounded case of premature optimization.